gui/dist_view/distviewcompute.h

```diff
@@ -132,6 +141,8 @@ struct ViewportCtx {
 
 	/* metadata depending on display configuration */
 	int nbins;
+
+	bool coloringUpToDate;
```

Comment:
please call coloringValid and rename "valid" to "metadataValid", and group them together.